afl/Cargo.toml

```
@@ -11,6 +11,7 @@ description = "Fuzzing Rust code with american-fuzzy-lop"
 repository = "https://github.com/rust-fuzz/afl.rs"
 homepage = "https://github.com/rust-fuzz/afl.rs"
 edition = "2021"
+include = ["README.md", "build.rs", "src/**/*.rs", "examples/**/*.rs"]
```

```suggestion
exclude = ["etc"]
```
I am afraid the `include` would cause something to break down the road, and that I would forget it was there.

Could I impose on you to make the same change here? https://github.com/rust-fuzz/afl.rs/blob/924fe86f34b7e15a224a1c5011ea7550cc24d7bb/cargo-afl/Cargo.toml#L14-L15

---

Sure, no problem

I've pushed an update changing that.

I personally prefer the explicit include syntax as it also prevents accidentally including other stuff that happens to be around while doing a cargo publish.